### PR TITLE
glossary: apply <abbr> tags to 'workspace' term et. al.

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Related to (but doesn't fully fix) #461

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Up to this point the PR does exactly what the title says.

Further commits will contain other misc. changes I didn't have time to finish in the previous "regular updates" PR.

File: src/Documentation/glossary.js [hunk 1]
```diff
@@ -8,16 +8,25 @@ export default {
...
       desc: `
-By "workspace" we refer to the directory containing all your project files. For
-example raw datasets, source code, ML models, etc. A workspace becomes a DVC
-project when [\`dvc init\`](/doc/commands-reference/init) is run, and
-[DVC-files](/doc/user-guide/dvc-file-format) are created in it. It\s typically
-also a Git repository.
+Directory containing all your project files. For example raw datasets, source
+code, ML models, etc. A workspace becomes a **DVC project** when
+[\`dvc init\`](/doc/commands-reference/init) is run, and
+[DVC-files](/doc/user-guide/dvc-file-format) are created in it.
```
Comment (shcheklein) [inline src/Documentation/glossary.js]: probably better to point to https://dvc.org/doc/user-guide/dvc-files-and-directories

File: src/Documentation/glossary.js [hunk 2]
```diff
@@ -8,16 +8,25 @@ export default {
...
+      `
+    },
+    {
+      name: 'DVC Project',
+      match: ['DVC project', 'project', 'projects'],
+      desc: `
+Initialized by running \`dvc init\` in the **workspace**. It will contain the
+\`.dvc/\` directory and [DVC-files](/doc/user-guide/dvc-file-format) created
```
Comment (shcheklein) [inline src/Documentation/glossary.js]: same here, this link **https://dvc.org/doc/user-guide/dvc-files-and-directories** is better I think

File: static/docs/commands-reference/fetch.md [hunk 3]
```diff
@@ -196,8 +198,10 @@ by all DVC-files in the current branch, including for directories. The checksums
 `3863d0e317dee0a55c4e59d2ec0eef33` and `42c7025fc0edeb174069280d17add2d4`
 correspond to the `model.pkl` file and `data/features/` directory, respectively.
 
+Let's link files from local cache to the workspace with:
+
 ```dvc
-$ dvc checkout      <- links files from local cache to workspace
+$ dvc checkout
 Checking out '{'scheme': 'local', 'path': '.../example-get-started/model.pkl'}' with cache '3863d0e317dee0a55c4e59d2ec0eef33'.
```
Comment (shcheklein) [inline static/docs/commands-reference/fetch.md]: not for this PR - we should revisit and update output here

File: static/docs/use-cases/multiple-data-scientists-on-a-single-machine.md [hunk 4]
```diff
@@ -33,17 +33,17 @@ permissions.
...
-need to simply move it from an old cache location to the new one:
+the shared cache directory (external to your workspace), you will need to simply
+move it from an old cache location to the new one:
 
 ```dvc
 $ mv .dvc/cache/* /path/to/dvc-cache
 ```
 
-### Configure External Cache
+### Configure External Cache (Optional)
```
Comment (shcheklein) [inline static/docs/use-cases/multiple-data-scientists-on-a-single-machine.md]: I've removed it optionality of this. It's not an optional step for this guide.

Comment (shcheklein) [reply static/docs/use-cases/multiple-data-scientists-on-a-single-machine.md]: it's better to use `Configure Shared Cache`

File: static/docs/use-cases/data-and-model-files-versioning.md [hunk 5]
```diff
@@ -23,9 +23,9 @@ ML <abbr>data artifacts</abbr> like data files, models, etc. Unlike `git-lfs`,
 DVC doesn't require installing a server; it can be used on-premises (NAS, SSH,
 for example) or with any major cloud provider (S3, Google Cloud, Azure).
 
-Let's say you already have a project that uses a bunch of images that are stored
-in `images` directory and has a `model.pkl` file - your model file that is
-deployed to production.
+Let's say you already have a <abbr>project</abbr> that uses a bunch of images
+stored in `images/` directory and has a `model.pkl` file - your model file that
```
Comment (shcheklein) [inline static/docs/use-cases/data-and-model-files-versioning.md]: do we actually need to specify `/` after dir names? is it some common practice?

File: static/docs/commands-reference/checkout.md [hunk 6]
```diff
@@ -122,7 +124,7 @@ $ cd example-get-started
 
 </details>
 
-The workspace looks almost like in this
+The workspace `tree` looks almost like in this
```
Comment (shcheklein) [inline static/docs/commands-reference/checkout.md]: I'm not sure we need to mention this explicitly tbh

File: src/Documentation/glossary.js (hunk 1 above)
Comment (jorgeorpinel) [reply src/Documentation/glossary.js]: Hm. This refers to DVC-files e.g. `stage.dvc` a.k.a. stage files. Not DVC files (and directories). Are you sure?

Comment (jorgeorpinel) [reply src/Documentation/glossary.js]: p.s. Added " or stage files" in 9206bdf to clarify, for now.

File: src/Documentation/glossary.js (hunk 2 above)
Comment (jorgeorpinel) [reply src/Documentation/glossary.js]: Linked the `.dvc/ directory` part to this in 9206bdf. Is it clearer now?

File: static/docs/commands-reference/fetch.md (hunk 3 above)
Comment (jorgeorpinel) [reply static/docs/commands-reference/fetch.md]: Oh yeah. Lots of code samples need to be double checked... All maybe? Opened #551 for this discussion.

File: static/docs/use-cases/multiple-data-scientists-on-a-single-machine.md (hunk 4 above)
Comment (jorgeorpinel) [reply static/docs/use-cases/multiple-data-scientists-on-a-single-machine.md]: Fixed in 0307e74

File: static/docs/use-cases/data-and-model-files-versioning.md (hunk 5 above)
Comment (jorgeorpinel) [reply static/docs/use-cases/data-and-model-files-versioning.md]: I do it only in relative directory names that don't include other slashes `/` to make it extra clear that it's a dir and not a file. Thanks for reminding me actually, I just made it consistent throughout docs in 8650a84 but lmk if you'd prefer all file and dir names to NOT have `/` at the end.

File: static/docs/commands-reference/checkout.md (hunk 6 above)
Comment (jorgeorpinel) [reply static/docs/commands-reference/checkout.md]: Removed in e0e0285.

File: src/Documentation/glossary.js (hunk 1 above)
Comment (shcheklein) [reply src/Documentation/glossary.js]: 👍

**Pull request merged**